src/modules/checkTask.js

```diff
@@ -0,0 +1,12 @@
+import { checkLocalTodos } from './localStorage.js';
+
+const checkTodo = (e) => {
+  const item = e.target;
+  if (item.classList[0] === 'complete-btn') {
+    const todo = item.parentElement.parentElement;
+    const todoIndex = todo.children[1].firstChild;
+    todoIndex.classList.toggle('completed');
+    checkLocalTodos(todo);
+  }
```

Comment:
- Kindly make sure that if you check a task as complete and refresh the page the check box appears marked as complete. 
 
![image](https://user-images.githubusercontent.com/98363075/199857350-9bbe64aa-a4a2-4da4-ad8c-783919a5918b.png)

- [OPTIONAL] Please ensure when you have tasks with the same name and you check a task just that task is updated in the local storage as complete and not the other tasks. 
![image](https://user-images.githubusercontent.com/98363075/199857558-b82e8756-331d-436c-8696-997602f3abc1.png)